app/models/auth_sources/auth_source_ldap.rb

```diff
@@ -27,6 +27,7 @@ class AuthSourceLdap < AuthSource
   include Parameterizable::ByIdName
   include Encryptable
   encrypts :account_password
+  include Taxonomix
```

Comment:
for some reason I was under impression it was listed among other LDAP auth sources, but looking at the code now proves me wrong => please ignore this